### DISCUSSION TITLE
汽車中控面板用大螢幕真的比較好嗎？

### DISCUSSION THREAD

**Discussion by tonyYZL** [1101-GS4719-C/Python-Community-GS4719-C#1188] [General]
隨著科技進步，汽車中控有一塊觸控螢幕可以說是標配了，而近幾年來，越來越多車款直接放一塊十幾寸的大螢幕來取代大部分的實體按鍵。

一個大螢幕裝在車上看起來很先進、很炫，但是它的實用性及方便性是一個值得深思的問題。

就我觀察身邊的人的開車習慣，我發現觸控螢幕對駕駛人來說很不方便操作，尤其在高速行駛時，駕駛的視線必須專注在前方，這導致他幾乎沒有辦法使用中控台。假如一台車配的都是實體按鍵，駕駛可以用觸覺的不同去找到他想要按的按鍵位置（類似電腦打字的感覺），這樣不但提升了安全性，也真正地達到中控台之所以存在的目的。

（我個人覺得大螢幕還有一個不算缺點小問題，就是晚上開車的時候很刺眼，而且調整的流程很繁瑣。不過這部分應該可以藉由軟體來改善）

當然，大螢幕也是有好處的。設計好的 UI 在視覺上更美觀，操作起來也比較人性化，顯示的資訊也更豐富、更有彈性。
簡單說就是一個字，潮。

如果我是乘客的話，我會選擇配大螢幕的車，但如果我是駕駛，這就要認真思考一下了。
想問問大家有什麼看法嗎？如果是你會怎麼選擇呢？

**Comment by zilinng** [comment]
我會選擇大螢幕的車，因為我覺得電子面板比起傳統按鈕能夠提供更加多元的功能，像是和手機連動，直接在車子的螢幕上顯示導航內容，或是播放音樂、影片等，車機結合AI的360度環景系統在我停車的時候也提供了非常大的幫助，隨著人工智慧不斷的發展，相信未來會有比這個更加方便的功能面市，可以提供駕駛更多輔助。至於安全性的部分，我認為加入聲控功能，讓駕駛可以在視線不離開前方的狀況下控制車子的空調、後視鏡或許是一個可以嘗試的解決方法。

**Comment by DuxtinTai** [comment]
分享一個，上禮拜日去企博才第一次看到的酷玩意兒
https://www.masters.tw/267904/holographic-imagery
透過在擋風玻璃上投影導航等，方便駕駛人不需要低頭看螢幕就可以得知道路訊息
比起按鈕或是觸控螢幕，我會更prefer投影在擋風玻璃（當然還是要可以看到前面的路），因為我真的是那種沒有導航就會迷路的大路癡

**Reply by tonyYZL** [reply]
投影導航還真的是第一次聽到！而且可以投影在整個前擋，超酷！以前只看過小小的 HUD

這樣的投影方式也許可以大幅改善現在導航語音的缺點，不過這個成本感覺就是貴貴貴，希望未來這個技術可以普遍應用在汽車上

**Comment by Lipunpun** [comment]
如果是我我會買大螢幕的欸，不管是導航、聽音樂、看影片，生活品質一個上升，就算是貸款200萬下去買，大螢幕讓我沈陷其中，真的爽。

**Comment by CCello** [comment]
設計師需要在美觀與實用性之間取得平衡。我認為，大螢幕是未來的發展方向，但是在保證安全性的前提下，設計師需要繼續優化螢幕上的 UI 和 UX，使其更加人性化、簡單易用、並且能夠減少駕駛人員的操作負擔，這樣才能真正發揮螢幕的優勢。在車上配備觸控螢幕是一個不可逆轉的趨勢，但我們應該繼續探索如何使它更好地服務於我們的生活。

**Comment by YoYooHuang** [comment]
我覺得大螢幕是很好的一個選擇，但關於作者提到的安全性，也是需要考慮到的。我覺得可行的解決方案是，在UI設計時，應該加入一些安全保護措施，例如：行駛中不能對螢幕做任何操作，除非有兩個以上的人在車上，我覺得這些都是可以加入設計思考的。

**Comment by Jack-1D** [comment]
我覺得如果現在市場流行大螢幕的車子，那麼以前那種有實體按鍵的車子應該會被時代逐漸淘汰。如果講求美觀，多數人仍然認為大螢幕比較闊氣。就如同現在手機很多都做成全螢幕一樣，有瀏海的手機也逐漸被淘汰了。

**Comment by brian431** [comment]
我也認為實體按鍵具有其存在的必要性。在開車時按按鈕最好是要非常直覺不需要多看一眼，不過現在的趨勢顯然是往大螢幕走。不知道有沒有什麼結合兩者優點的方法？個人認為一些正在開車時常常會使用到的操作還是實體的比較好。若是那種通常停下來才會使用到的再使用大螢幕來操作。
